src/libs/NextStepUtils.ts

```diff
@@ -216,16 +232,10 @@ function buildNextStep(report: OnyxEntry<Report>, predictedNextStatus: ValueOf<t
         }
 
         // Generates an optimistic nextStep once a report has been closed for example in the case of Submit and Close approval flow
+        // or when a report has been paid
         case CONST.REPORT.STATUS_NUM.CLOSED:
-            optimisticNextStep = {
```

Comment:
Can we add `optimisticNextStep = noActionRequired` and the `break`. I think it would improve readability a little.